### DISCUSSION TITLE
Session Timeout: How to extend?

### DISCUSSION THREAD

**Discussion by dhhdev** [directus/directus#5441] [Archive]
So, I have a use case. Where I will have the dashboard open in my browser, but it will timeout the session at some point. So if I try to update a record that has a session that has timed out, it will obviously fail. Since I will have to refresh and press continue my log in session. Loosing what I typed previously.

Is their a “keep alive” variable, or something? Perhaps just a matter of editing a config file?

P.S: Link to discord seems to be broken. (Says invalid/link is no longer valid.)

**Comment by rijkvanzanten** [comment]
I think this might be https://github.com/directus/directus/issues/4700 and/or https://github.com/directus/directus/issues/3940! You could increase the `ACCESS_TOKEN_TTL` value (https://docs.directus.io/reference/environment-variables/#security) as a temporary workaround (though the app should auto-refresh that no matter what)

> P.S: Link to discord seems to be broken. (Says invalid/link is no longer valid.)

What link where? https://directus.chat seems to be live as expected still 🙂

**Reply by dhhdev** [reply]
Regading the Discord link, I was regering the link from the README on the Directus GitHub Repo. If you click the link to join Discord it will give you the invalid link error.

Thanks for the tempoary workaround.

I might elaborate, that I am creating/editing a record in a collection. If I let my phone have its browser open. (Many hours.) It will eventually timeout, but it doesn’t say it. Before I try to update/save the record. And it gives me a “forbidden” error. Until I refresh the entire page and click the continue button, to “re-login”.
